utils/utils.py

```diff
@@ -116,11 +109,13 @@ def gpu_stats(device=0):
     Provides GPU utilization (%) and RAM usage
     :return: res.gpu, res.memory
     """
-    nvmlInit()
-    handle = nvmlDeviceGetHandleByIndex(device)
-    res = nvmlDeviceGetUtilizationRates(handle)
-    mem = nvmlDeviceGetMemoryInfo(handle)
-
+    torch.cuda.init()
+    res = {'gpu': torch.cuda.utilization(device)}
```

Comment:
![image](https://user-images.githubusercontent.com/24499815/193706942-96235d09-9532-456f-9db3-18c6d1245986.png)

Pynvml dependency is used in torch an error arises in a new environment without pynvml installed

Reply:
Weird since the test whit the environment without `pynvml` pass, this need an other investigation.

Reply:
> Weird since the test whit the environment without `pynvml` pass, this need an other investigation.

Probably what @valhassan was saying: if using torch with cpu, it passes, but with gpu it doesn't

Reply:
True I will need to check that